src/main/java/domain/DeckGenerator.java

```diff
@@ -0,0 +1,30 @@
+package domain;
+
+import domain.card.Card;
+import domain.card.Number;
+import domain.card.Shape;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class DeckGenerator {
+    private final List<Card> cards;
+
+    public DeckGenerator() {
+        cards = Arrays.stream(Shape.values())
+                .flatMap(this::makeCardsByShape)
+                .collect(Collectors.toList());
```

Comment:
```suggestion
        this.cards = Arrays.stream(Shape.values())
                .flatMap(this::makeCardsByShape)
                .collect(Collectors.toList());
```
(반영하지 않으셔도 됩니다.) 생성자에서는 this를 붙여주는 것이 좀 더 명시적이긴해요.

Reply:
동의합니다!! 놓쳤던 부분이네요..ㅎㅎ 꼼꼼하게 검수하겠습니다.